tests/fixtures/index.js

```diff
@@ -132,5 +132,8 @@ function loadFixture(path) {
     "./BAD_REQUIRE_INVALID_CALLBACK",
     "./BAD_REQUIRE_INVALID_ERRBACK",
     "./UNWRAPPED_FILE",
-    "./UNWRAPPED_FILE_NO_EXPRESSIONSTATEMENT"
+
+    // Edge cases that are actually valid
+    "./UNWRAPPED_FILE_NO_EXPRESSIONSTATEMENT",
```

Comment:
Ah crap, the description above is wrong. UNWRAPPED_FILE_NO_EXPRESSIONSTATEMENT is invalid. I'll fix and push hopefully in the next day or so. Don't merge this yet.